go/avatars/urlcaching.go

```diff
@@ -53,24 +53,24 @@ func (c *URLCachingSource) isStale(item lru.DiskLRUEntry) bool {
 }
 
 func (c *URLCachingSource) monitorAppState() {
-	c.debug(context.Background(), "monitorAppState: starting up")
+	m := libkb.NewMetaContextBackground(c.G())
+	c.debug(m, "monitorAppState: starting up")
 	state := keybase1.AppState_FOREGROUND
 	for {
 		state = <-c.G().AppState.NextUpdate(&state)
-		ctx := context.Background()
```

Comment:
@mmaxim  is there a good reason to make a new context everytime through this loop?  I couldn't figure it out

Reply:
No